tests/runner/src/lib.rs

```diff
@@ -105,6 +108,8 @@ where
         "-display",
         "none",
         "--no-reboot",
+        "-smp",
+        "8",
```

Comment:
Blog os at least doesn't support more than 1 core.

Reply:
Not sure why that matters here. This is just for the tests in this repo and has no impact on any kernel build with this bootloader. Also correct me if I am wrong but Blog Os should work fine on a multicore system. It will use just 1, but the other cores don't do anything unless specifically started by the kernel.